### PR TITLE
Updated Support for 24h Wordclock

### PR DESCRIPTION
**Pull request by dbambus**
- Added full support for Uhr_291
- deleted unused variables
- restructured uhr_func.h for setting up to date clock to matrix
- moved related enum declaration to Uhrtype.hpp

### CODE REVIEW

File: include/Uhr.h [hunk 1]
```diff
@@ -125,12 +125,6 @@ uint16_t wetterswitch;
 // LDR
 uint8_t ldrVal = 100;
 
-// Telnet vars
-bool ConnectionEstablished; // Flag for successfully handled connection
-#define MAX_TELNET_CLIENTS 2
-WiFiServer TelnetServer(23);
-WiFiClient TelnetClient[MAX_TELNET_CLIENTS];
-
```
Comment (tali) [inline include/Uhr.h:133]: How is this related to a 24h clock?
Please split the commit.

File: src/Wortuhr.ino [hunk 2]
```diff
@@ -85,9 +85,13 @@ NeoPixelBus<NeoBrgFeature, Neo800KbpsMethod> *strip_RGB = NULL;
...
 
 WiFiClient client;
-WiFiUDP ntpUDP;
 PubSubClient mqttClient(client);
 
+//--OTA--
+ESP8266WebServer httpServer(81);
+ESP8266HTTPUpdateServer httpUpdater;
+//--OTA--
+
```
Comment (tali) [inline src/Wortuhr.ino:94]: dito, please split into own commit.

Comment (dbambus) [reply src/Wortuhr.ino:94]: I ll do that.

File: include/Uhr.h (hunk 1 above)
Comment (dbambus) [reply include/Uhr.h:133]: I ll do that.
